uk.php

```diff
@@ -0,0 +1,406 @@
+  'tt--date-and-time' => '$[day] $[month.monthname.genitive], $[time]',
+  'tt--date-year-and-time' => '$[day] $[month.monthname.genitive] $[year], $[time]',
+
+  'tt--zone-pt' => 'Тихоокеанський час',
+  'tt--zone-mt' => 'Гірський час',
+  'tt--zone-ct' => 'Центральний час',
+  'tt--zone-et' => 'Східний час',
+  'tt--zone-gmt' => 'Час за Гринвічем',
+  'tt--zone-cet' => 'Центрально-європейский час',
+  'tt--zone-eet' => 'Східно-європейский час',
+  'tt--zone-msk' => 'Московский час',
+  'tt--zone-ekt' => 'Челябінський час',
+  'gs--timezone-offset-hours' => 'г',
+  'gs--timezone-offset-minutes' => 'хв',
+  
+  // mail
+  'em--comment-new-to-author-subject' => '$[commenter] коментує $[note-title]',
+  'em--comment-new-to-public-subject' => '$[commenter] коментує $[note-title]',
+  'em--comment-reply-to-public-subject' => '$[blog-author] відповідає на коментар',
+  'em--comment-reply' => '$[note-title] ($[blog-author] відповів)',
+  'em--created-automatically' => 'Листа створено автоматчно.',
+  'em--unsubscribe' => 'Відписатися від обговорення',
+  'em--reply' => 'Відповісти',
+  'em--comment-replied-at' => 'Коментар, на який відповів автор',
+  
+  // rss
+  'nf--comments-on-this-post' => 'Коментар до цього допису',
+  'gs--comments-on-post' => 'коментарі до допису',
+  'gs--comment-on-post' => 'коментар до допису',
+  'gs--posts-tagged' => 'дописи з тегом',
+  'gs--search-results' => 'результати пошуку',
+  
+  // social networks
+  'sn--twitter-verb' => 'Твітнути',
+  'sn--facebook-verb' => 'Поділитися',
+  'sn--gplus-verb' => 'Плюсанути',
+  'sn--vkontakte-verb' => 'Поділитися',
+  'sn--telegram-verb' => 'Надіслати',
+  'sn--pinterest-verb' => 'Запінити',
+
+  // umacros
+  'um--month' => '$[month.monthname]',
+  'um--month-short' => '$[month.monthname.short]',
+  'um--month-g' => '$[month.monthname.genitive]',
+  
+  // more strings
+  'gs--no-such-notes' => 'Таких дописів немає.',
+  'pt--page-not-found' => 'Сторінку не знайдено',
+  'gs--page-not-found' => 'Сторінку не знайдено.',
+  
+  'er--cannot-find-db' => 'Не можу знайти базу даних',
+  'er--cannot-connect-to-db' => 'Не можу зʼєднатися з базою даних',
+  'er--error-in-query' => 'Помилка під час запиту',
+  'er--error-occurred' => 'Сталася помилка',
+  'er--too-many-errors' => 'Надто багато помилок',
+  'gs--rss' => 'РСС',
+  
+  'gs--pgt' => 'Час генерації',
+  'gs--seconds-contraction' => 'с',
+  'gs--updated-successfully' => 'Виконано оновлення з версії $[from] до версії $[to]',
+  'gs--good-blogs' => 'Хороші блоги та сайти',
+  
+  );
+  
+}
+
+
+
+function e2lstr_monthname ($number, $modifier = '') {
+  if ($modifier == 'genitive') {
+    $tmp = array (
+      'грудня', 'січня', 'лютого', 'березня', 'квітня', 'травня', 'червня',
+      'липня', 'серпня', 'вересня', 'жовтня', 'листопада', 'грудня', 'січня'
+    );
+  } elseif ($modifier == 'short') {
+    $tmp = array (
+      'груд.', 'січ.', 'лют.', 'берез.', 'квіт.', 'трав.', 'черв.',

```

Comment:
Я за трёхбуквенный без точки. Конечно, без контекста может быть тяжело
сразу понять, причем тут "січ", "лис" или "сер", но в *данном* случае это
простейшее решение проблемы.
Кстати, помню именно такие сокращения (но с прописной буквы) на некоторых
карманных календариках, которые были очень популярны еще в 90-х.

Пт, 15 лип. 2016 19:56 користувач chimzuk <notifications@github.com> пише:

> In uk.php
> <https://github.com/ilyabirman/e2-languages/pull/2#discussion_r71004892>:
>
> > +
> > +  );
> > +
> > +}
> > +
> > +
> > +
> > +function e2lstr_monthname ($number, $modifier = '') {
> > +  if ($modifier == 'genitive') {
> > +    $tmp = array (
> > +      'грудня', 'січня', 'лютого', 'березня', 'квітня', 'травня', 'червня',
> > +      'липня', 'серпня', 'вересня', 'жовтня', 'листопада', 'грудня', 'січня'
> > +    );
> > +  } elseif ($modifier == 'short') {
> > +    $tmp = array (
> > +      'груд.', 'січ.', 'лют.', 'берез.', 'квіт.', 'трав.', 'черв.',
>
> @MustangUA <https://github.com/MustangUA> По-поводу «верес» и других, за
> основу брал правила сокращения.
> <http://www.library.ukma.edu.ua/fileadmin/documents/Bibliography/26_DCTU3582-97.pdf>
>
> правильні скорочення місяців року:
> січ., лют., берез., квіт., трав., черв., лип., серп., верес, жовт.,
> листоп., груд.;
>
> Думаю что правила сокращения выше работают для слов вне контекста, чтобы
> не возникало путаницы.
> «Гру» звучит не лучше чем «Лис», но понять месяц можно. Я не вижу проблем
> с использованием трёхбуквенных сокращений, если @ilyabirman
> <https://github.com/ilyabirman> говорит, что это желательно для
> интерфейса.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ilyabirman/e2-languages/pull/2/files/b304c3f563a7ae8efb84320195153be6390599a9#r71004892>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKR7xoChzwpLHuMLnFHDDuwCqRPwRi5aks5qV7uhgaJpZM4JI0vz>
> .
>
-- 

Vitali Bondar